enterprise/frontend/src/embedding-sdk/types/refresh-token.ts

```diff
@@ -3,6 +3,10 @@ export type MetabaseEmbeddingSessionToken = {
   exp: number;
 };
 
+export type UserBackendResponse = {
```

Comment:
```suggestion
export type UserBackendJwtResponse = {
```

so it's a bit more specific